src/genie/ui/dialogs/color_maps/black_blue_white.json

```diff
@@ -0,0 +1,33 @@
+[
```

Comment:
Keep only subset of paraview color maps:
viridis
cold hot
cold to warm
jet
linear green
rainbow desauturated